src/test/java/bio/terra/service/dataset/DatasetConnectedTest.java

```diff
@@ -104,10 +107,19 @@ public void setup() throws Exception {
             .name(Names.randomizeName(datasetRequest.getName()))
             .defaultProfileId(billingProfile.getId());
         summaryModel = connectedOperations.createDataset(datasetRequest);
+        logger.info("--------begin test---------");
     }
 
     @After
     public void tearDown() throws Exception {
+        logger.info("--------start of tear down---------");
+
+        // make sure all faults are set back to false
+        configService.setFault(ConfigEnum.FILE_INGEST_EXCLUSIVE_LOCK_FATAL_FAULT.toString(), false);
```

Comment:
The configService.reset will reset all of those faults. I think what we want is to have the reset happen before the teardown.

Reply:
Ahh, ok! That makes sense - thank you!